src/main/java/seedu/address/logic/commands/addcommand/AddMentorCommand.java

```diff
@@ -71,11 +71,15 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_SUCCESS, this.mentor.toString()), PrefixType.M);
     }
 
+    @Override
```

Comment:
Or you can use getClass()

Reply:
@hcwong sorry can you clarify what you think should be declared as a static final variable?

Reply:
@john0227 yup you're right. But I implemented the toString() method for each Command class because I thought maybe we could discuss what we wanted to be displayed in the GUI for the command history. So I thought I could implement a toString() method first. But yeah, if the current implementation is deemed suitable, then the getClass() method would do great

Reply:
Okay, the changes have been made, as per our telegram convo :)